sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -348,8 +348,37 @@ def estimate_size(self, unused_value, nested=False):
     return 8
 
 
-class TimestampCoderImpl(StreamCoderImpl):
+class IntervalWindowCoderImpl(StreamCoderImpl):
+  # TODO: Fn Harness only supports millis. Is this important enough to fix?
+  def _to_normal_time(self, value):
+    """Convert "lexicographically ordered unsigned" to signed."""
+    return value - 0x8000000000000000
```

Comment:
`(1 << 63)` for better readability. It should only be computed once and stored in the bytecode.